### ISSUE TITLE
NSIS is required for build

### ISSUE THREAD

**Issue by holmnikolaj** [mavlink/qgc-dev-guide#57]
QGC can not compile without NSIS in windows. This would be a nice information in the "getting started" section. 
It can be downloaded at:
https://sourceforge.net/projects/nsis/

**Comment by DonLakeFlyer** [comment]
That should only be required if you are building an installer. For the majority of users it is not needed. Are you finding that to not be true?

**Comment by holmnikolaj** [comment]
Yes I am. However for those who wants an installer it would be helpful information.

**Comment by DonLakeFlyer** [comment]
@hamishwillee I don't think there are any docs on building an installer for QGC. That is done by adding CONFIG+=installer to the qmake build. And on Window syou need to install NSIS.

**Comment by hamishwillee** [comment]
@DonLakeFlyer No their aren't. My QT is rusty - to do this you'd just update [qgroundcontrol.pro](https://github.com/mavlink/qgroundcontrol/blob/master/qgroundcontrol.pro#L1273) as shown right?
I'm STILL downloading Qt to test this.

```
...
# Installer targets
#

CONFIG+=installer

include(QGCInstaller.pri)
```

**Comment by DonLakeFlyer** [comment]
You change it in Qt Creator: Projects -> Build -> Build Steps -> qmake -> Additional arguments
![screen shot 2019-01-18 at 10 14 10 am](https://user-images.githubusercontent.com/5876851/51404974-d6911a00-1b09-11e9-8e41-bcda26675229.png)

**Comment by holmnikolaj** [comment]
Vrv

Få Outlook til Android<https://aka.ms/ghei36>

________________________________

Venlig hilsen / Best regards,
Nikolaj Stjerne Holm
Embedded Control Engineer
+45 9686 7666 | (+45) 2367 0550
nh@sky-watch.com | www.sky-watch.com
Østre Allé 6F- 9530 Støvring-Denmark
